### DISCUSSION TITLE
Page categories blank after JRE update

### DISCUSSION THREAD

**Discussion by mariosm1** [percussion/percussioncms#846] [Troubleshooting]
**Describe the bug**
Categories do not appear in page metadata after the JRE install used by the cms is updated while the system is running.   Restarting the cms fixes the categories.

**To Reproduce**
1. Add categories to the CMS under Administration > Categories
2. While the system is running, replace the JRE install in the Percussion/JRE folder, or update it with a package manager if it's sym linked
3. Wait a little while then edit a page
4. Go to Actions > Edit Metadata
5. Open the Categories view.  Categories should be blank
<img width="682" alt="Bildschirm­foto 2023-05-17 um 12 31 46 PM" src="https://github.com/percussion/percussioncms/assets/32443983/9a329645-0365-47f7-935c-4f905c19963b">

**Expected behavior**
Updating JRE should not affect the Page Metadata categories.

**Comment by natechadwick** [comment]
@matomario8 When the system JRE is updated the CMS service will always need to be restarted if the <InstallDir>/JRE is linked to a system JRE.

**Comment by natechadwick** [comment]
If they are using yum, a tool like needs-restarting from yum-utils https://man7.org/linux/man-pages/man1/needs-restarting.1.html  can list services that need restarting after an update.
